src/stylesheets/components/_section.scss

```diff
@@ -1,37 +1,37 @@
 .usa-section {
-  @include u-padding-y($site-margins);
+  @include u-padding-y(units(4));
```

Comment:
We have a v2 var for this one.
You can use `u-padding-y($theme-site-margins)` here.

Reply:
This is on me, I recommended the units here bc I don't think there's a correlation between the site margins and the section component. I think it's fine as is.